policy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+	require_once("template.php");
+	require_once("config.php");
+	Template::render_header("Privacy Policy");
+
+	echo "<h1>" . _("Privcacy Policy") . "</h2>";
+	echo "<h2>" . _("Who we are") . "</h2>";
+	echo WHO_WE_ARE;
+	echo "<h2>" . _("Contact") . "</h2>";
+	echo POLICY_NAME . "<br>";
+	echo ADDRESS . "<br>";
+	echo POLICY_MAIL . "<br>";
+	echo POLICY_PHONE . "<br>";
+
+	echo '<h2>' . _("What personal data we collect and why") . '</h2>';
+	echo '<h3>' . _("Global") . "</h3>";
+	echo _("If you access our websites, the following information will be saved: IP-address, Date, Time, Browser queries, 
+	General information about your browser, operating system and all search queries on the sites. 
+	This user data will be used for anonym user statistics to recognize trends and improve our content.
+	") . "<br><br>";
+	echo "<h2>" . _("How we protect your data") . "</h2>";
+	echo _("In collaboration with our hosting provider we try our best to protect our 
+	databases against access from third parties, losses, misuse or forgery.
+	") . "<br><br>";
+	echo "<h2>" . _("Third party that receive your personal data") . "</h2>";
+	echo "Our hosting provider can access the date we store on their server. We have a data processing agreement with them.";
+	echo "<h3>" . _("Cookies") . "</h3>";
+	echo _("This site uses cookies – small text files that are placed on your machine to help the site provide a better user experience. 
+	In general, cookies are used to retain user preferences, store information for things like shopping carts, 
+	and provide anonymised tracking data to third party applications like Google Analytics. 
```

Comment:
Maybe the anonymised tracking data isn't necessary in this case? I'll leave this up to you.

Reply:
If we add Google Analytics (like mentioned in #16 ) this needs to be in the policy (and tracking data needs to be anonymised).